editor/src/clj/editor/localization.clj

```diff
@@ -613,10 +615,36 @@
   (let [message-localizable (message-pattern? message)]
     (->Transform (cond-> message (not message-localizable) str) message-localizable f args)))
 
+(defn annotate-as-sorted
+  "Annotate a vector of items as sorted with localization-aware sorting function
+
+  Args:
+    f        localization-aware sorting function, receive localization state,
+             items, and the rest of the arguments; see [[natural-sort-by-label]]
+    items    vector of items to annotate"
+  [f items]
+  {:pre [(vector? items)]}
+  (vary-meta items assoc ::sort f))
+
+(defn natural-sort-by-label
+  "Localization-aware function that sorts the items on their :label vals"
+  [localization-state items]
+  (->> items
+       (mapv #(coll/pair (localization-state (:label %)) %))
```

Comment:
Should we `refer` `pair`?

Reply:
I'd prefer if we didn't `refer` things that can be required/aliased instead. Perhaps with the exception of `clojure.test`.